src/annotator/plugin/test/pdf-metadata-test.js

```diff
@@ -42,23 +42,42 @@ class FakePDFViewerApplication {
   /**
    * Initialize the "PDF viewer" as it would be when loading a document or
    * when a document fails to load.
+   *
+   * @param {string} url - Fake PDF URL
```

Comment:
Thank you very much for adding these thorough comments. This could be crazymaking in the future without them.